packages/init/bootstrap-starter/near-runner/__tests__/main.ava.ts

```diff
@@ -0,0 +1,162 @@
+  const contract = await root.createAndDeploy(
+    // Subaccount name
+    'status-message',
+
+    // Relative path (from package.json location) to the compiled contract file
+    // which will be deployed to this account
+    'compiled-contracts/status_message.wasm',
+
+    // Provide `method` and `args` to call in the same block as the deploy
+    // {
+    //   method: 'init',
+    //   args: {owner_id: root},
+    // },
+  );
+
+  // You can call any other contract methods that you want to be executed at the
+  // beginning of all subsequent tests. In this example, Alice sets her status.
+  // Don't forget to `await` your calls!
+  await alice.call(contract, 'set_status', {message: 'hello'});
+
+  // Return the accounts that you want available in subsequent tests
+  // (`root` is always available)
+  return {alice, contract};
+});
+
+/**
+ * Now you can write some tests! In local sandbox mode, each `runner.test` will:
+ *
+ *   - start a new local blockchain
+ *   - copy the state from the blockchain created in `Runner.create`
+ *   - get access to the accounts created in `Runner.create` using the same variable names
+ *   - run concurrently with all other `runner.test` calls, keeping data isolated
+ *   - shut down at the end, forgetting all new data created
+ *
+ * It's also worth knowing that `runner.test` is syntax sugar added by
+ * near-runner-ava. With raw AVA + near-runner, here's how to write a test:
+ *
+ *     import avaTest from 'ava';
+ *     import {Runner} from 'near-runner';
+ *     // Alternatively, you can import Runner and ava both from near-runner-ava:
+ *     // import {ava as avaTest, Runner} from 'near-runner-ava';
+ *
+ *     const runner = Runner.create(...);
+ *
+ *     avaTest('root sets status', async test => {
+ *       await runner.run(async ({contract, root}) => {
+ *         ...
+ *       });
+ *     });
+ *
+ * Instead, with the syntax sugar, you can write this as you see it below –
+ * saving an indentation level and avoiding one extra `await`.
+ * (Extra credit: try rewriting this test using the "sugar-free" syntax.)
+*/
+runner.test('root sets status', async (test, {contract, root}) => {
+  // Don't forget to `await` your calls!
+  await root.call(contract, 'set_status', {message: 'lol'});
+
+  // Assert that two things are identical using `test.is`
+  test.is(
+    // Note that Root called the contract with `root.call(contract, ...)`, but
+    // you view the contract with `contract.view`, since the account doing the
+    // viewing is irrelevant.
+    await contract.view('get_status', {account_id: root}),
+    'lol',
+  );
+});
+
+runner.test('statuses initialized in Runner.create', async (test, {alice, contract, root}) => {
+  // If you want to store a `view` in a local variable, you can inform
+  // TypeScript what sort of return value you expect.
+  const aliceStatus: string = await contract.view('get_status', {account_id: alice});
```

Comment:
```suggestion
  const aliceStatus = await contract.view('get_status', {account_id: alice}) as string;
```
It's better to cast the type.

Reply:
Why is it better?

Reply:
I find it more declarative since the action you are doing is a cast.  I have also had issues with the former in the past.

Reply:
Right now we have issues with the `const aliceStatus: string` syntax because we type `contract.view` as `string | any` I think. But that seems like a bug in our `view` typing. If it supported the declaration syntax rather than only the coercion syntax, which style do you think would be better?

I am under the impression that the declaration syntax can catch problems sooner that the coercion syntax obfuscates until runtime, and that people should prefer the declaration syntax where possible.

Reply:
They both work, but declaration syntax usually catching problem's soon since you are resolving the type of the expression before even considering the wider context.  However, It's really problem to keep with the syntax if you really believe strongly. 

Austin had found it confusing using the declaration because he had assumed that meant that variable would be a string at runtime rather than only seen as one during compile time.

Reply:
> Austin had found it confusing using the declaration because he had assumed that meant that variable would be a string at runtime rather than only seen as one during compile time.

This is true with either approach, unfortunately.

I've updated the code to show both approaches, since we don't seem to have a strong recommendation either way.